plugins/mta/src/components/ApplicationInventory  /ApplicationInventory.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import useAsync from 'react-use/lib/useAsync';
+
+import { Table } from '@backstage/core-components';
+import { configApiRef, useApi } from '@backstage/core-plugin-api';
+
+import { Box, Grid, Typography } from '@material-ui/core';
+
+import { Columns, Data } from './tableHeading';
+
+export const ApplicationInventory = () => {
+  const config = useApi(configApiRef);
+  const BACKEND_URL = 'backend.baseUrl';
+
+  const { loading: isSysInfoLoading, value: sysInfoData } =
+    useAsync(async (): Promise<Data> => {
+      const backendUrl = config.getString(BACKEND_URL);
+      const backendApiEndPoint = `${backendUrl}/api/<>/<>`;
```

Comment:
@redhatHameed Is the backend endpoint (after the `backendUrl`) actually `/api/<>/<>`?

Reply:
updated

However, this might change as the API client needs to be written, also added comments